pkg/azuredisk/azure_test.go

```diff
@@ -71,12 +71,12 @@ users:
 		{
 			desc:        "[failure] out of cluster, no kubeconfig, no credential file",
 			kubeconfig:  "",
-			expectedErr: fmt.Errorf("Failed to load config from file: %s, cloud not get azure cloud provider", DefaultCredFilePath),
```

Comment:
I changed test case `TestGetCloudProvider`, pls make sure the test case change is valid, thanks.
@ZeroMagic

Reply:
the failure is caused by `/etc/kubernetes/azure.json` on my machine.